Allow elmish style tuples (#10186)
dotnet/docs: docs/fsharp/style-guide/formatting.md

| x, y -> 1
```

> It is qlso accepted to omit parentheses if the tuple is the return value of a function:

Reviewer: ```suggestion
It is also commonly accepted to omit parentheses if the tuple is the return value of a function:
```

```fsharp
// OK

> let f model msg =

Reviewer: ```suggestion
let update model msg =
```

Just so that folks coming here from Fable know it immediately

    match msg with
    | 1 -> model + 1, []
    | _ -> model, [ msg ]
```

> 

Reviewer: ```suggestion
In summary, prefer parenthesized tuple instantiations, but when using tuples for pattern matching or a return value, it is considered fine to avoid parentheses.
```
